aws/dns.go

```diff
@@ -0,0 +1,10 @@
+package aws
+
+import "net"
+
+func reverseLookup(ip string) (name string) {
+	if names, err := net.LookupAddr(ip); err == nil && len(names) > 0 {
+		name = names[0]
```

Comment:
Although this is valid Go, I think normal idiom in this case would be to use an unnamed return value and have multiple `return` statements:

```go
if ... {
    return names[0]
}
return ""
```

This makes the result in the unsuccessful case more obvious. Named return values are usually used only where the name adds useful context in package documentation for functions with multiple return values of the same type.

This is discussed in [the common Go code review comments](https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters). This is of course a subjective thing, so feel free to ignore this if you'd made a conscious decision here.

Reply:
👍